cli/rt/11_url.js

```diff
@@ -218,13 +218,16 @@
       yield* this.#params;
     }
 
+    // need to handle special case for `+`
+    // ref: https://url.spec.whatwg.org/#concept-urlencoded-serializer
     toString() {
       return this.#params
         .map(
           (tuple) =>
             `${encodeURIComponent(tuple[0])}=${encodeURIComponent(tuple[1])}`,
         )
-        .join("&");
+        .join("&")
+        .replaceAll("%20", "+");
```

Comment:
This is not the only difference between the current behaviour and the spec...

Maybe this should be left to those who claimed the issue.